<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RasterFromLiDARUI</class>
 <widget class="QWidget" name="RasterFromLiDARUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Raster from LiDAR</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_input">
     <property name="title">
      <string>Dossier d'entrée</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_input">
      <item>
       <widget class="QLineEdit" name="lineEdit_input"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_input">
        <property name="text">
         <string>Parcourir</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_output_mnt">
     <property name="title">
      <string>Dossier de sortie MNT</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_output_mnt">
      <item>
       <widget class="QLineEdit" name="lineEdit_output_mnt"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_output_mnt">
        <property name="text">
         <string>Parcourir</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_output_ombrage">
     <property name="title">
      <string>Dossier de sortie Ombrage</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_output_ombrage">
      <item>
       <widget class="QLineEdit" name="lineEdit_output_ombrage"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_output_hillshade">
        <property name="text">
         <string>Parcourir</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_fillnodata">
     <property name="text">
      <string>Remplir les valeurs nulles</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_fill">
     <item>
      <widget class="QSpinBox" name="spinBox_distance">
       <property name="prefix">
        <string>Distance: </string>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_iterations">
       <property name="prefix">
        <string>Itérations: </string>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_classification">
     <property name="title">
      <string>Filtrage par classification</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_classification">
      <item>
       <widget class="QLineEdit" name="lineEdit_champ_class">
        <property name="placeholderText">
         <string>Nom du champ classification</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_valeurs_class">
        <property name="placeholderText">
         <string>Valeurs (ex: 2,3,4)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_raster_params">
     <item>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_resolution">
       <property name="prefix">
        <string>Résolution: </string>
       </property>
       <property name="maximum">
        <double>10.000000000000000</double>
       </property>
       <property name="value">
        <double>0.500000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_tile">
       <property name="prefix">
        <string>Taille tuile: </string>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="value">
        <number>256</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_sun">
     <item>
      <widget class="QSpinBox" name="spinBox_azimuth">
       <property name="prefix">
        <string>Azimut: </string>
       </property>
       <property name="maximum">
        <number>360</number>
       </property>
       <property name="value">
        <number>315</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_angle">
       <property name="prefix">
        <string>Angle: </string>
       </property>
       <property name="maximum">
        <number>90</number>
       </property>
       <property name="value">
        <number>45</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_zfactor">
       <property name="prefix">
        <string>Z factor: </string>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_run">
     <property name="text">
      <string>Lancer le traitement</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
